ui/pl.json

```diff
@@ -0,0 +1,24 @@
+{
+  "building-delete": "Usuń",
+  "building-move": "Przesuń",
+  "contact_completed": "Kontrakt został wypełniony.",
```

Comment:
Little nitpicking but it's not *the contract* but *a contract has been unlocked*. I don't know if there's a difference in polish but the translator gave me the wrong one :)

Reply:
No there's no difference. We don't have an equivalent for the word "the"